flag_timestamp.go

```diff
@@ -11,7 +11,10 @@ type TimestampFlag = FlagBase[time.Time, TimestampConfig, timestampValue]
 // TimestampConfig defines the config for timestamp flags
 type TimestampConfig struct {
 	Timezone *time.Location
-	Layouts  []string
+	// Available layouts for flag value.
+	//
+	// Note that value for formats with missing year/date will be interpreted as current year/date respectively.
+	Layouts []string
```

Comment:
I'd appreciate an example of what is a correct layout, or a link to docs.

Reply:
Definition of a correct layout stays the same (comparing to main) - it is such a string `lo`, that causes no error calling `time.Parse(lo, val)`, where `val` is time, formatted same as `lo`.
Valid format: `15:04`
Invalid format: `11:51`
Playground example: https://go.dev/play/p/EW82fbXJdPg
More about timestamp layouts [HERE](https://pkg.go.dev/time#pkg-constants)

Reply:
Please include this in the doc comment.

Reply:
Done